torch/testing/_asserts.py

```diff
@@ -102,6 +97,63 @@ def wrapper(actual: Tensor, expected: Tensor, **kwargs: Any) -> Optional[Excepti
     return wrapper
 
 
+def _check_quantization(check_tensor_values: Callable[..., Optional[Exception]]) -> Callable[..., Optional[Exception]]:
+    """Decorates continuous tensor check functions to handle quantized tensors.
+
+    If the inputs are not quantized, this decorator is a no-op.
+
+    Args:
+        check_tensor_values (Callable[..., Optional[Exception]]): Tensor check function for continuous tensors.
+
+    Returns:
+        Optional[Exception]: Return value of :attr:`check_tensors`.
+    """
+
+    @functools.wraps(check_tensor_values)
+    def wrapper(actual: Tensor, expected: Tensor, **kwargs: Any) -> Optional[Exception]:
+        if not actual.is_quantized:
```

Comment:
In this expected would be quantized... how does that work?

Reply:
Nope. This decorates `_check_values_(equal|close)` which is used in `_check_tensors_(equal|close)` after `_check_attributes_equal` was called. This rules out that only one input is quantized. See L207-L208.